src/core/server/opensearch/version_check/ensure_opensearch_version.ts

```diff
@@ -36,16 +36,44 @@
  */
 
 import { timer, of, from, Observable } from 'rxjs';
-import { map, distinctUntilChanged, catchError, exhaustMap } from 'rxjs/operators';
+import { map, distinctUntilChanged, catchError, exhaustMap, mergeMap } from 'rxjs/operators';
+import { get, forEach } from 'lodash';
 import {
   opensearchVersionCompatibleWithOpenSearchDashboards,
   opensearchVersionEqualsOpenSearchDashboards,
 } from './opensearch_opensearch_dashboards_version_compatability';
 import { Logger } from '../../logging';
 import type { OpenSearchClient } from '../client';
 
+export const getNodeId = async (internalClient: OpenSearchClient): Promise<string | null> => {
+  try {
+    const state = await internalClient.cluster.state({
+      filter_path: ['nodes.*.attributes.cluster_id'],
```

Comment:
Is there a more standardized way of defining names of attributes instead of using strings everywhere?

Reply:
I think the attributes have to be specified as strings